cli/helpers/downloader.go

```diff
@@ -0,0 +1,26 @@
+package helpers
```

Comment:
This was used to download the eirini release but we changed that to be an embedded tarball. I kept this around because we could use it to fetch our own dependencies if needed (e.g. download a specific version of `kubectl` and `helm` if they don't exist locally and after asking the user).